src/components/BorrowerProfile/LendCta.vue

```diff
@@ -222,20 +227,32 @@
 						All shares reserved
 					</p>
 					<hr
-						class="tw-hidden md:tw-block tw-border-tertiary tw-w-full tw-my-2"
+						class="lg:tw-block tw-border-tertiary tw-w-full tw-my-2"
+						:class="[
+							{
+								'tw-hidden': isSticky,
+								'tw-block': !isSticky,
+							}
+						]"
 					>
 					<div
-						class="tw-flex lg:tw-justify-center tw-w-full"
-						:class="isLoggedIn ? 'tw-justify-between' : 'tw-justify-end'"
+						class="lg:tw-block tw-flex lg:tw-justify-center tw-w-full"
+						:class="[
+							{
+								'tw-justify-between': isLoggedIn,
+								'tw-justify-end': !isLoggedIn,
+								'tw-hidden': isSticky,
+							}
+						]"
 					>
 						<loan-bookmark
 							v-if="isLoggedIn"
 							data-testid="bp-lend-cta-loan-bookmark"
 							:loan-id="loanId"
-							class="tw-hidden md:tw-inline-block lg:tw-hidden"
+							class="tw-inline-block lg:tw-hidden"
```

Comment:
It still seems we're just arbitrarily hiding the bookmark function on desktop... not sure why but I think this was supposed to be something temporary due to other features being added.

Reply:
@eddieferrer wasn't that part of the new share button work? Can we show this on desktop now that this PR also makes the share-button visible at the same code point.

Reply:
Yes

Reply:
It makes sense to me to not hide this at all. From our last team chat it seemed clear that the intent for the bookmark button is to always display it for logged-in users.